crates/cuid1/src/cuid1/fingerprint.rs

```diff
@@ -34,13 +35,16 @@ thread_local! {
         [
             thread_rng().gen::<u128>().to_be_bytes(),
             thread_rng().gen::<u128>().to_be_bytes(),
+            #[cfg(not(target_arch="wasm32"))]
```

Comment:
This mean the process id and thread id do not contribute to the entropy of the hash. I'm not sure about the implications given these entropies are hashed. in any case it seems not substantially less resistant to RNG attacks.

Reply:
Yes. I am not super happy about it either. For future we should provide some alternative. Browser version of JS cuid uses hash of all globals, for example.

Created a follow-up issue - https://github.com/prisma/team-orm/issues/630

Reply:
Imho `cuid` should straight-up be removed from Prisma in Prisma 6, and we should use `cuid2` instead to avoid these entropy problems altogether.